gulpfile.js

```diff
@@ -353,8 +352,12 @@ function deployCesium(bucketName, uploadDirectory, cacheControl) {
     var totalFiles = 0;
     var uploaded = 0;
     var skipped = 0;
+    var errors = [];
 
     return listAll(s3, bucketName, uploadDirectory, existingBlobs)
+    .catch(function(error) {
```

Comment:
If `listAll` fails, I'm pretty sure we would just want to avoid the entire operation (which was the old behavior) but this catch makes that no longer the case.